packages/ui/src/form/Label/Label.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames';
+import React, { FunctionComponent, LabelHTMLAttributes } from 'react';
+
+export interface LabelProps extends LabelHTMLAttributes<HTMLLabelElement> {
+    testId?: string;
+    hidden?: boolean;
+}
+
+const Label: FunctionComponent<LabelProps> = ({ children, className, hidden, testId, ...rest }) => (
+    <label
+        {...rest}
+        className={classNames(
+            className || 'form-label',
+            { 'is-srOnly': hidden },
+            'optimizedCheckout-form-label',
+        )}
+        data-test={testId}
+    >
+        {children}
+    </label>
+);
+
+export default Label;
```

Comment:
This will still be bundled as part of building core package, SCSS files live in core and are built as part of it there. This UI package will be source of UI components going forward. I haven't given much thought to scss as of now. Maybe can be next iteration.